itests/src/test/resources/testconfiguration.properties

```diff
@@ -237,7 +237,6 @@ mr.query.files=\
   inputwherefalse.q,\
   join_map_ppr.q,\
   join_vc.q,\
-  list_bucket_dml_8.q,\
```

Comment:
Why remove this?

Reply:
If a qfile is not contained in this list, then TestMiniLlapLocalCliDriver(the default dirver of qtest) pickes it.